code/modules/mob/language.dm

```diff
@@ -570,4 +570,28 @@
 	else
 		return ..()
 
+/datum/language/human/monkey
+	name = "Chimpanzee"
+	desc = "Ook ook ook."
+	speech_verb = "chimpers"
+	ask_verb = "chimpers"
+	exclaim_verb = "screeches"
+	key = "6"
```

Comment:
word of warning; all of these keys are already taken up already, so I'm not sure how well this will work out or not.